src/lightning/fabric/strategies/fsdp.py

```diff
@@ -97,7 +84,6 @@ def __init__(
         process_group_backend: Optional[str] = None,
         timeout: Optional[timedelta] = default_pg_timeout,
         cpu_offload: Union[bool, "CPUOffload", None] = None,
-        backward_prefetch: Optional["BackwardPrefetch"] = None,
```

Comment:
I didn't see why this argument is here and not passed through `kwargs` as with the rest, fine with reverting if this was intended.